src/ClassEmitter.ts

```diff
@@ -99,7 +99,8 @@ export class ClassEmitter {
 		var hasDirectChildren = 
 			classObject.properties.length > 0 || 
 			classObject.methods.length > 0 || 
-			classObject.fields.length > 0;
+			classObject.fields.length > 0 ||
+			classObject.inheritsFrom.length > 0;
```

Comment:
Can we call it extends?

Also, classes can also have implementations. Can we add support for that too?